Update/omot_005_choice2.txt

```diff
@@ -248,7 +248,7 @@ void main()
 //rもし…万が一幸運にも赤坂が大石と組めばこれ以上なく力強いのだが、現場叩き上げの大石だけでは、鷹野たちの組織だった陰謀に立ち向かうのは難しいだろう。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "もし…万が一幸運にも赤坂が大石と組めばこれ以上なく力強いのだが、現場叩き上げの大石だけでは、鷹野たちの組織だった陰謀に立ち向かうのは難しいだろう。",
-			NULL, "If... Akasaka were able to team up with Ooishi they'd be even more powerful. But it would be difficult for Ooishi to stand up to Takano and her organization on his own.", Line_Normal);
+			NULL, "If... Akasaka were able to team up with Ooishi they'd be even more powerful. But Ooishi's career has been limited to on-site things so it would be difficult for him to stand up to Takano and her organization on his own.", Line_Normal);
```

Comment:
```suggestion
			NULL, "If... Akasaka were able to team up with Ooishi they'd be even more powerful. But Ooishi was just an on-site worker so it would be difficult for him to stand up to Takano and her organization on his own.", Line_Normal);
```